src/java/org/apache/cassandra/db/ReadCommand.java

```diff
@@ -658,6 +658,9 @@ public UnfilteredRowIterator applyToPartition(UnfilteredRowIterator iter)
             @Override
             protected Row applyToStatic(Row row)
             {
+                if (row == Rows.EMPTY_STATIC_ROW)
```

Comment:
This was incorrectly counted and brought the size over the limit for `LocalReadSizeWarningTest`.

Reply:
+1